src/main/java/com/cloudbees/jenkins/support/filter/ContentMapping.java

```diff
@@ -49,32 +48,45 @@
     private static final String ALT_SEPARATOR = " » ";
 
     private final String original;
-    private final Pattern pattern;
```

Comment:
Could you elaborate? I can't understand clearly what you say

Reply:
I think @jvz refers to the fact that these content mappings are stored into disk. If you change the format now, you should make them work with the previous format too.

Reply:
Exactly. These are persisted, partially so that it can be configured. Now I'm sort of wondering how important it is to persist these.

Reply:
AFAIK the `ContentMapping` is not persisted. The one persisted is `com.cloudbees.jenkins.support.filter.ContentMappings.xml`. It only persists the original and replacement strings. Perhaps I'm wrong, you've worked much more than me on this plugin.

Reply:
Not exactly. The ContentMappings in the end contains instances of ContentMapping, but when saving/reading from/to disk only the original and replacement fields are taken into account.

Reply:
As agreed on the channel, it's fine because the ContentMapping already has the SerializationProxy class with the `readResolve` method used in the `writeReplace` method.